packages/rainbowkit/src/wallets/getDefaultWallets.ts

```diff
@@ -28,6 +29,7 @@ export const getDefaultWallets = ({
         metaMaskWallet({ chains }),
         walletConnectWallet({ chains }),
         braveWallet({ chains }),
+        bitskiWallet({ chains }),
```

Comment:
Revert addition here. Add to `packages/example` implementation for QA test deployment.

Reply:
The reason we added this was because if it's not there by default, then the Injected Wallet is what shows instead. We were hoping it would work like Web3Onboard, where if the user has Bitski installed it would show up without needing to manually configured. Is that not the case?

Reply:
@pzuraq That's the expected behavior currently. We plan on rolling out 2 things soon:
- alterations to the `injectedWallet` connector so that the metadata defaults to the connector for the actively injected wallets, rather than just using fallback metadata. We are also evaluating forcing this connector as a default in the `Custom Wallet List` scenario, where dApps could have historically not included `injectedWallet`. This should make RainbowKit work better with things like the Gnosis Safe browser, as well as preserving the goals of the `Custom Wallet List`.
- a new community wallet list project that RainbowKit inherits so that Rainbow isn't the only arbiter of wallet popularity

Reply:
Ah, that makes perfect sense! Sounds good, I've updated the PR per your feedback so it should be ready to go now. Let us know if there's anything we can help out with for the updates to the `injectedWallet` connector!